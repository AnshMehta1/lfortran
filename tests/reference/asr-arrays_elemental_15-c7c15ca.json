{
    "basename": "asr-arrays_elemental_15-c7c15ca",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_elemental_15.f90",
    "infile_hash": "8104c3cedadd8d2c591c7e64d6c6eab01bc45aa998dbec8d6fe4876c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-arrays_elemental_15-c7c15ca.stdout",
    "stdout_hash": "bd65606683e59c0784557c43a56332636159b4da8add82539c6359f4",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}