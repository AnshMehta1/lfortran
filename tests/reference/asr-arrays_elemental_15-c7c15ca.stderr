LCOMPILERS_ASSERT failed: $DIR/src/libasr/asr_utils.h
function type_get_past_array(), line number 198 at 
!ASR::is_a<ASR::Array_t>(*e->m_type)
Aborted (core dumped)
